RELEASE.md

```diff
@@ -0,0 +1,18 @@
+Release type: minor
+
+Support for schema-extensions in subscriptions.
+
+i.e:
+```python
+class MyExtension(SchemaExtension):
+
+    async def on_operation(self, execution_context):
+        #  This part is called before the subscription starts
+        yield
+        #  This part is called after the subscription ends
+
+    async def on_execute(self, execution_context):
+        #  This part is called before a new result is coming from the subscription
+        yield
+        #  This part is called after the subscription yields a result
```

Comment:
this runs for all operations, right? I wonder if we should explain how to differentiate between queries/mutations and subscriptions

Reply:
Yeah I'll add a section in the docs